docs/release_notes/v0.9-alpha.md

```diff
@@ -0,0 +1,37 @@
+# Spice v0.9-alpha (Mar 14, 2024)
+
+The v0.9 release adds several data connectors including the Spice data connector for the ability to connect to other `spiced` instances. Improved observability for `spiced` has been added with the new `/metrics` endpoint for monitoring deployed instances.
+
+## Highlights in v0.9-alpha
+
+**Spice.ai data connector**: Use other `spiced` instances as data connectors for federated SQL query across `spiced` deployments and for chaining `spiced` deployments.
+
+**Keyring secret store**: Use the operating system native credential store, like macOS keychain for storing secrets used by `spiced`.
+
+**PostgreSQL data connector**: [PostgreSQL](https://www.postgresql.org/) can now be used as both a data store for acceleration and as a connector for federated SQL query.
+
+**Databricks data connector**: [Databricks](https://www.databricks.com) as a connector for federated SQL query across Delta Lake tables.
+
+**Metrics endpoint**: Added new `/metrics` endpoint for `spiced` observability and monitoring.
+
```

Comment:
Is it worth calling out s3 support too since our goal is to show we support as much as possible?

Reply:
Done!